src/main/kotlin/io/github/zap/build/gradle/convention/ProjectExtension.kt

```diff
@@ -169,4 +171,99 @@ fun Project.configureShadow(destination: File,
     tasks.named("build", DefaultTask::class.java) {
         it.dependsOn(sj)
     }
+}
+
+fun Project.qs(name: String = "", options: ExternalModuleDependency.() -> Unit = {}) : ExternalModuleDependency.() -> Unit {
```

Comment:
```suggestion
fun Project.qs(
    name: String = "",
    options: ExternalModuleDependency.() -> Unit = {}
): ExternalModuleDependency.() -> Unit {
```
https://developer.android.com/kotlin/style-guide Functions section, pretty sure this is a de facto standard

Reply:
bro reference without navigation code z. Anyway, other languages do that too, but the method doesn't that long

Reply:
it goes past the gray IntelliJ line 👿

Reply:
then?

Reply:
just 4 characters

Reply:
it's just nicer to look at and follows convention

Reply:
It still follows the convention, it downs to how do you define "long"